index.html

```diff
@@ -93,14 +95,16 @@ <h5 class="connect">LET’S CONNECT</h5>
           <li>
             <img
               src="img/card Portfolio.png"
-              alt="card"
+              alt
+              text="nature card"
```

Comment:
- [ ] Good job adding the `alt` attribute to your images 👏  unfortunately, this is not the correct syntax to implement an alt text to an image, it should be:
```
 <img src="img/card Portfolio.png" class="desktop-images" alt="Tonic website screenshot">
```
- [ ] You should consider double-checking all the images and their `alt` attribute in the project, to guarantee they all have the correct syntax and meaningful text.
> For more info on alt text, you read [this article](https://axesslab.com/alt-texts/).